src/CheckoutSdk/Payments/Customer.cs

```diff
@@ -1,9 +0,0 @@
-namespace Checkout.Payments
-{
-    public class Customer
```

Comment:
The PaymentRequest.Customer and CustomerSource have two different responsibilities, so this should not be removed. The former is to provide customer details on the root of the payment request, the latter represents a payment source. The moment that these two types diverge (as they will since we need to add Cvv to customer sources) we start sending irrelevant data on the request.

Reply:
At the time these objects' properties were virtually the same so I figured I'll divide them once they diverge because I wasn't sure it will happen. Thanks for pointing this out.